tests/tiny_server.py

```diff
@@ -0,0 +1,12 @@
+import json
```

Comment:
This issue has been cleared too since I removed the flask dependency and tiny_server.py